src/cache/DefaultCache.php

```diff
@@ -53,7 +66,7 @@ public function set(string $name, $value)
    * @param  string $ip_address IP address to lookup in cache.
    * @return mixed IP address data.
    */
-    public function get(string $name)
+    public function get(string $key, callable $callback, ?float $beta = null, ?array &$metadata = null): mixed
```

Comment:
Why is it accepting all these params? It's changing the interface and won't work.

Reply:
`
PHP Fatal error:  Declaration of ipinfo\ipinfo\cache\DefaultCache::get(string $name) must be compatible with Symfony\Contracts\Cache\CacheInterface::get(string $key, callable $callback, ?float $beta = null, ?array &$metadata = null): mixed in /home/mslm/ipinfo/php/src/cache/DefaultCache.php on line 68
`

I get this error that it must be compatible with symfony cache interface.

Reply:
Is that because we are supposed to be implementing Symfony's cache interface or because we're returning with a `cache->get` that has that interface? Because this is our own interface we're implementing here, from `CacheInterface` in the same folder.